.pre-commit-config.yaml

```diff
@@ -15,17 +15,17 @@ repos:
       - id: end-of-file-fixer
       - id: trailing-whitespace
   - repo: https://github.com/psf/black
-    rev: 22.3.0
+    rev: 22.12.0
     hooks:
       - id: black
   - repo: https://github.com/pycqa/isort
     # isort config is in setup.cfg
-    rev: 5.9.3
+    rev: 5.11.4
```

Comment:
You'll need to bump isort to 5.12.0 or higher (both here and setup.py) befor merging because of https://github.com/PyCQA/isort/issues/2077

Reply:
done